plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -234,15 +234,14 @@ public void testSplittableNotCheckedOnSmallFiles()
         hiveSplitSource = hiveSplitSource(backgroundHiveSplitLoader);
         backgroundHiveSplitLoader.start(hiveSplitSource);
 
-        try {
-            drainSplits(hiveSplitSource);
-            fail("Expected split generation to call isSplittable and fail");
-        }
-        catch (TrinoException e) {
-            Throwable cause = Throwables.getRootCause(e);
-            assertTrue(cause instanceof IllegalStateException);
-            assertEquals(cause.getMessage(), "isSplittable called");
-        }
```

Comment:
in line 234, hiveSplitSource  is assigned and never changed after it. May be lines 234,235,237 can be changed to reduce code as followed: 
HiveSplitSource finalHiveSplitSource = hiveSplitSource(backgroundHiveSplitLoader);
        backgroundHiveSplitLoader.start(finalHiveSplitSource);

Reply:
@MariyaR I've refactored the code as suggested.